src/utils/getShadowWithColor.ts

```diff
@@ -0,0 +1,32 @@
+import { ShadowsType } from "../styles/shadows";
+import { getColorWithOpacity } from "./getColorWithOpacity";
+
+// shadow の色を theme.shadows で定義した色から変更したいときに使う
```

Comment:
```suggestion
/**
 * shadow の色を theme.shadows で定義した色から変更したいときに使う関数
 */
```

jsdoc記法だとエディタに優しい。

<img width="815" alt="image" src="https://user-images.githubusercontent.com/40020812/188419449-0aad746b-6dfa-4833-a7eb-db68be64d814.png">